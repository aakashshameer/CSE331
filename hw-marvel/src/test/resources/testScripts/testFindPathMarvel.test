# test BSF on marvel graph

# build graph
LoadGraph marvel marvel.csv

# check BFS
FindPath marvel M'SHULLA GORILLA-MAN