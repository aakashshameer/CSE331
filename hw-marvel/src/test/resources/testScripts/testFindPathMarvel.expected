# test BSF on marvel graph

# build graph
loaded graph marvel

# check BFS
path from M'SHULLA to GORILLA-MAN:
M'SHULLA to HAWK via AA2 19
HAWK to GORILLA-MAN via AVF 4